app/src/main/java/com/globant/practice/domain/interactor/UserInteractor.java

```diff
@@ -0,0 +1,36 @@
+package com.globant.practice.domain.interactor;
+
+import com.globant.practice.domain.model.User;
+import com.globant.practice.domain.service.GitHubApi;
+import java.util.List;
+import javax.inject.Inject;
+import io.reactivex.Observable;
+
+/**
+ * Provides all the methods necessary to manage the user information.
+ * Created by jonathan.vargas on 4/04/2017.
+ */
+
+public class UserInteractor {
```

Comment:
An interactor should be a class that implements Command Pattern. Also the name of the class must be self explanatory.